convene-web/app/javascript/src/application.scss

```diff
@@ -1,5 +1,9 @@
 h2 {
   @apply text-xl font-medium text-gray-900;
 }
+.center-container {
+  @apply flex flex-col items-center justify-center px-4 pt-28;
```

Comment:
This feels like a composite utility class; which is probably quite useful; but I would prefer we used the `--modifier` syntax for naming. Perhaps this is `.--centers-contents-vertically`